src/components/Task.vue

```diff
@@ -2,6 +2,7 @@
   <div
     v-bind:id="task.gid"
     class="task"
+    :class="singleTaskView ? 'single-task-view' : ''"
```

Comment:
This doesn't feel quite right. The task "shouldn't know" about whether it is a single task or not. That should be handled by the parent